src/components/NewUserForm.vue

```diff
@@ -1,26 +1,72 @@
 <template>
-    <section class='signup-form'>
+    <form @submit.prevent="handleSubmit" class='signup-form'>
         <b-field label="Name">
-            <b-input class='input-field' value=""></b-input>
+            <b-input class='input-field' required v-model="account.name" value="" maxlength="40" placeholder="Please enter your name"></b-input>
         </b-field>
 
         <b-field label="Email">
             <b-input class='input-field' type="email"
+                required
+                placeholder="Please enter your email"
+                v-model="account.email"
                 value=""
                 maxlength="40">
             </b-input>
         </b-field>
+        <b-field label="Proficiency" for="proficiency">
```

Comment:
There are two down arrows for the proficiency section; i've attached a screenshot below
<img width="651" alt="Screen Shot 2020-01-27 at 3 46 01 PM" src="https://user-images.githubusercontent.com/24424825/73221369-e4667b80-411d-11ea-8aa0-c4b265bd8c9d.png">

Reply:
Thank you! This has been fixed
<img width="1109" alt="Screen Shot 2020-01-31 at 6 05 53 PM" src="https://user-images.githubusercontent.com/49662828/73584310-b00ff980-4454-11ea-95c4-30aa8c36c431.png">

Reply:
Nice!